.semaphore/pre-release.yml

```diff
@@ -22,8 +22,18 @@ blocks:
       jobs:
         - name: "Publish to NPM"
           commands:
+            - git checkout -b pre-release-$SEMAPHORE_PIPELINE_ID
+            - git push origin pre-release-$SEMAPHORE_PIPELINE_ID --no-verify
```

Comment:
Checkout and do the publishing work on a clean branch, to avoid the error happening because of a PR just merged in during the publishing task.
I have to push this branch, otherwise lerna will complain about this https://github.com/lerna/lerna/blob/main/commands/version/lib/is-behind-upstream.js